test/categories/test_shapes.py

```diff
@@ -0,0 +1,20 @@
+import unittest
+from qual_id.categories.shapes import Shapes
+import random
+
+
+class TestFood(unittest.TestCase):
+  def setUp(self):
+    self.shapes = Shapes()
+
+  def test__get_values__returns_list(self):
```

Comment:
The way that unit tests are written has been simplified. If you pull the latest changes and take a look at another unit test

Reply:
This hasn't been resolved.